QUARKUS-5613.md

```diff
@@ -0,0 +1,83 @@
+## Links to the resources
+- JIRA ticket: https://issues.redhat.com/browse/QUARKUS-5613
+- GH PRs related to the JIRA issue: https://github.com/quarkusio/quarkus/pull/43283 https://github.com/quarkusio/quarkus/issues/42117
+- TP/TD PR: https://github.com/quarkus-qe/quarkus-test-plans/pull/193
+- Guides modified and related:
+  * https://quarkus.io/guides/security-openid-connect-client-reference
+  * https://quarkus.io/guides/security-oidc-code-flow-authentication
+  * https://quarkus.io/guides/security-openid-connect-client-registration
+
+## Scope of the testing
+
+Our main testing scope is to ensure that the new OIDC response filter mechanism works as intended for various OIDC endpoints, ensuring that is possible intercept and handle responses
+from various OIDC endpoints. This complements the already available OidcRequestFilter functionality.
+This kind of validation is possible by asserting that:
+    - response body can be read and serialized,
+    - status code is correct,
+    - response headers are correct
+    - and request properties matches the ones recorded in the request filter
+
+This scope will be covered on areas related to:
+   - Security - ensuring integration with OIDC provider (as Keycloak) plus Quarkus OIDC and OIDC Client extensions.
+   - Logging /Tracing -  to enable logs only at finer levels for our specific filter classes to confirm that the filter is triggered. This helps debug the response filter’s internal behavior.
+
+The test coverage will be verified for JVM, native mode, and Openshift.
+The testing include OIDC Client (quarkus-oidc), OIDC Client (quarkus-oidc-client) and OIDC client registration (quarkus-oidc-client-registration) extensions.
+
+### What to Test - Scenarios
+We are focusing on these three scenarios (Token,User info, Client Registration) as they represent the most common endpoints for Quarkus OIDC users.
```

Comment:
```suggestion
We are focusing on these three scenarios (Token, UserInfo, Client Registration) as they represent the most common endpoints for Quarkus OIDC users.
```

> most common endpoints

I am really no OIDC expert, so take it with reservations, but: I doubt that the client registration is most common. I think it is rather infrequent scenario. You are ignoring scenarios that are truly most common as JWKS calls ...

Reply:
Include JWKS scenario and dropping the client-registration scenario.